src/redux/initialState.tsx

```diff
@@ -1,12 +1,78 @@
-const initialState = {
-  kickscooters: [],
+import { ProductType } from "../types/ProductType";
+import { CartItem } from "../types/CartItem";
 
-  accessories: [],
+export interface Store {
+  kickscooters: {
+    id: string,
+    title: string,
+    price: number,
+    image: string,
+    isSelect: boolean,
+    inStock: number,
+    gift: string,
+    gallery: {
+      image: string
+    }[],
+    delivery: {
+      title: string,
+      subtitle: string,
+      text: string,
+      ico: string
+    }
+    warranty: {
+      title: string,
+      subtitle: string,
+      text: string,
+      years: string,
+      extend: {}
+    },
+    support: {
+      title: string,
+      subtitle: string,
+      text: string,
+      ico: string
+    },
+    description: {
+      title: string,
+      subtitle: string,
+      text: string,
+      miles: string
+    },
+    packingList: string,
+    youtubeVideo: string
+  }[];
 
-  cart: [],
+  accessories: {
+    id: string,
+    title: string,
+    description: string,
+    price: number,
+    image: string,
+    inStock: number
+  }[],
 
-  device: [],
+  cart?: CartItem[],
+
+  device?: {
+    isDesktop: boolean
+  }[],
```

Comment:
I don't think there are many devices in your app. Isn't there is only one `device: { isDesktop: boolean }`?